performance-timeline/po-disconnect.html

```diff
@@ -30,15 +30,16 @@
 
   async_test(function (t) {
     var observer = new PerformanceObserver(
-        t.step_func(function(entryList, obs) {
-          checkEntries(entryList.getEntries(),
-            [{ entryType: "mark", name: "mark1"}]);
-          t.done();
+        t.step_func(function (entryList, obs) {
+          assert_unreached("This callback must not be invoked");
```

Comment:
I don't believe it is plausible. The spec says to queue a task when the mark is created, which I believe is an asynchronous operation. By the time the task runs, the observer has cleared its individual buffer via disconnect, so it doesn't have anything to deliver. If tasks are not async operations, then may could be right.

See my complete diagnosis on: https://github.com/w3c/performance-timeline/issues/66